datapackage_pipelines_mojp/common/processors/sync.py

```diff
@@ -15,7 +15,8 @@
                                     {'name': 'id', 'type': 'string'},
                                     {"name": "version", "type": "string",
                                      "description": "source dependant field, used by sync process to detect document updates"},
-                                    {"name": "collection", "type": "string", "description": COLLECTION_FIELD_DESCRIPTION},
+                                    {"name": "collection", "type": "string",
+                                        "description": COLLECTION_FIELD_DESCRIPTION},
                                     {"name": "source_doc", "type": "object"},
                                     {"name": "title", "type": "object",
                                      "description": "languages other then he/en, will be flattened on elasticsearch to content_html_LANG"},
```

Comment:
how will this schema for after sync be used?

in the DBS_DOCS_TABLE_SCHEMA you need to add a "title_lc" field of type "object" which will contain each language and it's lower-case name - same as the existing "title" field